app/config/footer.json

```diff
@@ -0,0 +1,78 @@
+{
+	"socials": [
+		{
```

Comment:
Please add a link as the first position:
- Newsletter "https://www.gonative.cc/#newsletter"

Reply:
Done